docs/architecture/blazor-for-web-forms-developers/project-structure.md

```diff
@@ -1,9 +1,233 @@
 ---
-title: Project structure for ASP.NET Web Forms and Blazor apps
+title: Project structure for Blazor apps
 description: Comparing the project structure of ASP.NET Web Forms and Blazor projects.
 author: danroth27
 ms.author: daroth
 ms.date: 09/11/2019
 ---
 
-# Project structure for ASP.NET Web Forms and Blazor apps
+# Project structure for Blazor apps
+
+Blazor apps share a lot in common with ASP.NET Web Forms apps. Though the projects look quite different, the concepts have many similarities. Here we will look at the structure of a Blazor project and compare it to an ASP.NET Web Forms project.
+
+To create your first Blazor app, follow the instructions in the [Blazor getting started steps](https://docs.microsoft.com/aspnet/core/blazor/get-started). You can follow the instructions to create either a Blazor Server app or a Blazor WebAssembly app hosted in ASP.NET Core. Most of the code in both projects is the same. Although, the hosting model-specific logic will differ.
+
+## Project file
+
+Blazor Server apps are .NET Core projects. The project file for the Blazor Server app is about as simple as it can get:
+
+```xml
+<Project Sdk="Microsoft.NET.Sdk.Web">
+
+  <PropertyGroup>
+    <TargetFramework>netcoreapp3.0</TargetFramework>
+  </PropertyGroup>
+
+</Project>
+```
+
+The project file for a Blazor WebAssembly looks slightly more involved (exact version numbers may vary):
+
+```xml
+<Project Sdk="Microsoft.NET.Sdk.Web">
+
+  <PropertyGroup>
+    <TargetFramework>netstandard2.0</TargetFramework>
+    <OutputType>Exe</OutputType>
+    <LangVersion>7.3</LangVersion>
+    <RazorLangVersion>3.0</RazorLangVersion>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="Microsoft.AspNetCore.Blazor" Version="3.1.0" />
+    <PackageReference Include="Microsoft.AspNetCore.Blazor.Build" Version="3.1.0" PrivateAssets="all" />
+    <PackageReference Include="Microsoft.AspNetCore.Blazor.HttpClient" Version="3.1.0" />
+    <PackageReference Include="Microsoft.AspNetCore.Blazor.DevServer" Version="3.1.0" PrivateAssets="all" />
+  </ItemGroup>
+
+  <ItemGroup>
+    <ProjectReference Include="..\Shared\BlazorWebAssemblyApp1.Shared.csproj" />
+  </ItemGroup>
+
+</Project>
+```
+
+Blazor WebAssembly projects target .NET Standard instead of .NET Core because they run in the browser on a WebAssembly-based .NET runtime. And since you can't install .NET into a web browser like you can on a server or developer machine, the project references the Blazor framework using individual package references.
```

Comment:
```suggestion
Blazor WebAssembly projects target .NET Standard instead of .NET Core because they run in the browser on a WebAssembly-based .NET runtime. Since you can't install .NET into a web browser like you can on a server or developer machine, the project references the Blazor framework using individual package references.
```